### ISSUE TITLE
StartupException - can't start the embedded RabbitMQ

### ISSUE THREAD

**Issue by ctiliescu** [AlejandroRivera/embedded-rabbitmq#50]
The error received at run() phase new StartupHelper(config).call();

io.arivera.oss.embedded.rabbitmq.helpers.StartupException: Could not confirm RabbitMQ Server initialization completed successfully within 3000ms

    new EmbeddedRabbitMqConfig.Builder()
      .envVar(RabbitMqEnvVar.NODE_PORT, randomAvailablePort())
      .build()
    rabbitMq = new EmbeddedRabbitMq(config)

    try {
    rabbitMq.start()
    } catch {
      case e: Exception =>
    }

**Comment by AlejandroRivera** [comment]
@ctiliescu can you provide the full logs? 
The initialization is not considered completed until the RabbitMQ process prints out a message saying: `completed with X plugins`

For example:
```
21:05:43.477 INFO    Thread-3 i.a.o.e.r.E.P.rabbitmq-server -               RabbitMQ 3.6.6. Copyright (C) 2007-2016 Pivotal Software, Inc.
21:05:43.477 INFO    Thread-3 i.a.o.e.r.E.P.rabbitmq-server -   ##  ##      Licensed under the MPL.  See http://www.rabbitmq.com/
21:05:43.477 INFO    Thread-3 i.a.o.e.r.E.P.rabbitmq-server -   ##  ##
21:05:43.477 INFO    Thread-3 i.a.o.e.r.E.P.rabbitmq-server -   ##########  Logs: /var/folders/j6/4qlzsrl541q6nqyfhbkmfnvr0000gn/T/rabbitmq_server-3.6.6/var/log/rabbitmq/rabbit-test.log
21:05:43.477 INFO    Thread-3 i.a.o.e.r.E.P.rabbitmq-server -   ######  ##        /var/folders/j6/4qlzsrl541q6nqyfhbkmfnvr0000gn/T/rabbitmq_server-3.6.6/var/log/rabbitmq/rabbit-test-sasl.log
21:05:43.477 INFO    Thread-3 i.a.o.e.r.E.P.rabbitmq-server -   ##########
21:05:43.477 INFO    Thread-3 i.a.o.e.r.E.P.rabbitmq-server -               Starting broker...
21:05:44.265 INFO    Thread-3 i.a.o.e.r.E.P.rabbitmq-server -  completed with 0 plugins.```

**Comment by ctiliescu** [comment]
DEBUG [main] (MessageLoggers.java:42) - Executing [/var/folders/sy/hqr2nz653wl7nfm5wxmq8p6j7t2z0_/T/rabbitmq_server-3.6.9/sbin/rabbitmq-server] in /var/folders/sy/hqr2nz653wl7nfm5wxmq8p6j7t2z0_/T/rabbitmq_server-3.6.9.
DEBUG [main] (MessageLoggers.java:42) - Started java.lang.UNIXProcess@14acaea5
DEBUG [main] (LoggingProcessListener.java:29) - Process started.
 INFO [Thread-2] (Slf4jInfoOutputStream.java:35) - ERROR: node with name "rabbit" already running on "LBUXXXX"
DEBUG [WaitForProcess-java.lang.UNIXProcess@14acaea5] (MessageLoggers.java:42) - java.lang.UNIXProcess@14acaea5 stopped with exit code 1
DEBUG [WaitForProcess-java.lang.UNIXProcess@14acaea5] (LoggingProcessListener.java:37) - Process finished (exit code: 1).
DEBUG [WaitForProcess-java.lang.UNIXProcess@14acaea5] (StartupHelper.java:149) - No more output is expected since process finished (exit code: 1)
DEBUG [WaitForProcess-java.lang.UNIXProcess@14acaea5] (LoggingProcessListener.java:45) - Process stopped

After I enabled i found the problem, even if rabbitMQ was closed it was a confict between rabbitMQ node name and Embedded RabbitMQ.

Thanks!
